### PR TITLE
Compose EOL banner

### PR DESCRIPTION
**Pull request by aevesdocker**
Builds on https://github.com/docker/docs/pull/16350 and ties in nicely with https://github.com/docker/docs/pull/16521

### CODE REVIEW

File: _includes/compose-eol.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+> **Important**
+>
+> From April 2023 Compose V1 won't be supported anymore and will be removed from all Docker Desktop versions.
```
Comment (glours) [inline _includes/compose-eol.md:3]: @hyu don't we change the eol date to end of june?

**Review by glours — DISMISSED** [review]
We just need to check the eol date with @hyu, otherwise LTGM 👍

File: _includes/compose-eol.md (hunk 1 above)
Comment (aevesdocker) [reply _includes/compose-eol.md:3]: Silly me, this is my bad. Yes. EOL date has changed to June, I forgot to update

Comment (aevesdocker) [inline _includes/compose-eol.md:3]:
```suggestion
> From the end of June 2023 Compose V1 won't be supported anymore and will be removed from all Docker Desktop versions.
```

**Pull request closed without merging**